[DOCU-2314] Konnect Service Hub: UI validation and style cleanup (#4082)
Kong/docs.konghq.com: app/konnect/servicehub/manage-services.md

Access all {{site.konnect_short_name}} service configuration through the {% konnect_icon servicehub %}
**Service Hub**.

## Add a service to the catalog

1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), click the **New service** button.

1. Enter a display name.

    A display name can be any string containing letters, numbers, spaces, or the following

>     characters: `.`, `-`, `_`, `~`, or `:`. Spaces are equal to the `-` character.

Reviewer: ```suggestion
    characters: `.`, `-`, `_`, `~`, `:`. Spaces are equal to the `-` character.
```

    For example, you can use `example_service`, `ExampleService`, `Example-Service`, or `Example Service`.
    `Example-Service` and `Example Service` would be considered the same name.

    The display name you create generates a service name. {{site.konnect_short_name}}
    uses the service name for internal metadata.

1. Optional: Enter a description.

1. Click **Create**.

## Add labels to a service

1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.

1. In the header next to **Labels**, click **Edit**.

1. Click **Add label** and add any labels in `key:value` pair format.

    For example, you might set `location:us-west`, where `location` is the key
    and the `us-west` is the value.

>     These labels are completely custom. Set anything that you need.

Reviewer: ```suggestion
    These labels are customizable, you can set them to any value.
```
